doc/real_index.html

```diff
@@ -65,18 +65,6 @@ <h3>Installation</h3>
 </ul>
 
 
-<h3>Discussion</h3>
-
-<ul>
-<li>
-  <p><a class="reference external" target="_blank" href="https://lists.secondlife.com/cgi-bin/mailman/listinfo/eventletdev">eventletdev at lists.secondlife.com</a></p>
```

Comment:
We should maybe update the topic in IRC, too:
```
Eventlet discussion, ask away! | http://eventlet.net/ | Mailing list: http://urlenco.de/usbo |Latest release: http://pypi.python.org/pypi/eventlet
```
Looks like urlenco.de isn't a thing anymore?

Reply:
I have registered project channel and updated topic.

@jstasiak @nat-goodspeed or anybody who needs access to something related to eventlet channel on IRC, please write me.